src/wsk-tasks/sass.js

```diff
@@ -50,3 +50,7 @@ export function build() {
   return sassStream.pipe(sourcemaps.write('.'))
     .pipe(gulp.dest(GLOBAL.config.dest));
 }
+
+module.exports = {
```

Comment:
I think no to IIFE explicit export is kind of nice in this scenario (at least for now).

Any benefit to using UMD?